src/adsp_config.c

```diff
@@ -1023,8 +1037,10 @@ static int parse_signed_pkg(const toml_table_t *toml, struct parse_ctx *pctx,
 	/* bitmap array */
 	bitmap_array = toml_array_in(signed_pkg, "bitmap");
 	if (!bitmap_array) {
-		/* default value */
-		out->bitmap[4] = 8;
+		if (production)
+			out->bitmap[4] = 0x8;
```

Comment:
This is not related to prod bit, please revert all bitmap[] changes

Reply:
or you can add another commit to setup properly IMR type, there is already such option:

fprintf(stdout, "\t -i set IMR type\n");

so you can set bitmap to 0x10 if imr type = 4

Reply:
ok